tox.ini

```diff
@@ -20,6 +20,8 @@ ignore =
     FI51
     FI53
     FI54
+    W504
```

Comment:
Why are we excluding more warning types?

Reply:
New bad warning came up after bumping flake8 / pycodestyle, rolling back this bumping.

Reply:
Trying de-bump flake8 / pycodestyle